.tasks/docs.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+
+vars:
+  GOBIN: "{{.USER_WORKING_DIR}}/tools"
+  TFPLUGINDOCS_CMD: "{{.GOBIN}}/tfplugindocs"
+
+tasks:
+  install:
+    desc: "Install documentation tools"
+    cmds:
+      - echo "Installing tfplugindocs..."
+      - mkdir -p {{.GOBIN}}
+      - GOBIN={{.GOBIN}} go install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs@latest
+      - echo "tfplugindocs installed successfully to {{.GOBIN}}"
```

Comment:
I would suggest to add `status` to not perform installation every time the `task docs:install` is called.

https://taskfile.dev/docs/reference/schema#status